replicate-between-primary-and-secondary-clusters.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Replicate data between primary and secondary clusters
 summary: Learn how to replicate data from a primary cluster to a secondary cluster.
-aliases: ['/docs/dev/incremental-replication-between-clusters/']
+aliases: ['/docs/dev/incremental-replication-between-clusters/', '/tidb/stable/replicate-betwwen-primary-and-secondary-clusters/']
```

Comment:
```suggestion
aliases: ['/docs/dev/incremental-replication-between-clusters/', '/tidb/dev/replicate-betwwen-primary-and-secondary-clusters/']
```

Reply:
The server uses aliases to do redirects for deleted files. Therefore, **the value of the `aliases` field** depends on **the URL of the file you delete**. And **where to put the aliases** depends on **the page you'll redirect the users to**.

For example, if you delete `tidb-overview.md` from v6.0 branch (the url of which is `docs.pingcap.com/tidb/v6.0/tidb-overview`) and you hope users get redirected to the `tidb-architecture.md` file in the v6.1 branch, you need to add `/tidb/v6.0/tidb-overview` to the `tidb-architecture.md` file on release-v6.1 branch.

Reply:
Oh, forget to mention that master branch is mapped to `dev` on our docs site, and release-6.1 mapped to `stable.